core/src/main/java/com/linecorp/armeria/server/logging/DefaultAccessLoggerStrategy.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.server.logging;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.function.Function;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.linecorp.armeria.server.VirtualHost;
+
+/**
+ * .
+ */
+public final class DefaultAccessLoggerStrategy {
+    static final String ACCESS = "com.linecorp.armeria.logging.access";
+    static final Function<VirtualHost, Logger> instance = (host) -> LoggerFactory.getLogger(
+            ACCESS + "." + reverseName(host.hostnamePattern()));
```

Comment:
- `ACCESS + "."` could be simplified to `ACCESS` if we append `.` to the definition of `ACCESS`.
- Move the string concatenation into `reverseName()` so that the concatenation of `ACCESS` is done against `StringBuilder`. (efficiency)